src/styles/blocks/calendar.scss

```
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: left;
+  padding: $calendar-padding;
+  width: calc(7 * ($width-day + $gap-day) + 2 * $calendar-padding - $gap-day) ;
+  gap: $gap-day;
+
+  &__day {
+    display: flex;
+    position: relative;
+    width: $width-day;
+    height: 100px;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+
+    border: 1px solid #000;
```

don't forget to create variables for all colors
```suggestion
    border: 1px solid $c-black;
```